internal/cypress/config.go

```diff
@@ -200,3 +200,23 @@ func Validate(p Project) error {
 
 	return nil
 }
+
+// SplitSuites divided Suites to dockerSuites and sauceSuites
+func SplitSuites(p Project) (Project, Project) {
+	dockerSuites := []Suite{}
```

Comment:
Could you perhaps replace those with the more canonical approach?
```suggestion
	var dockerSuites []Suite
```

Reply:
sure.